symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/Message.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.core.service.message.model;
+
+import static java.util.Collections.emptyMap;
+
+import com.symphony.bdk.template.api.Template;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apiguardian.api.API;
+
+@Getter
+@Setter
+@Accessors(fluent = true)
+@API(status = API.Status.EXPERIMENTAL)
+public class Message {
```

Comment:
We also need the optional `version` field, default value is `2.0`.

Reply:
I don't really understand the purpose of `version` field here.

Reply:
That's the version of the `MessageML` format

Reply:
https://github.com/symphonyoss/messageml-utils/#usage